### PR TITLE
fix: search bar and contibutor dropdown added

### PR DESCRIPTION
**Pull request by rajnijain6**
# Title and Issue number 

Title :

Issue No. :

Code Stack : 

Close #<issue_no>

# Checklist:

- [] I have mentioned the issue number in my Pull Request.
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by sadanandpai** [comment]
image
breaks on multiple selection @rajnijain6

**Comment by jrajni** [comment]
Scroll added on y-axis, let me know if it doesn't not satisfy the requirement

File: apps/host/src/components/common/multi-select/multi-select.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          control: (base) => ({
+            ...base,
+            width: width,
+            height: '0.1rem',
+            textAlign: 'left',
+            outline: 'none',
+            borderRadius: '5px',
+            border: '2px solid #ccc',
+            maxHeight: '200px',
+            overflowY: 'scroll',
```
Comment (sadanandpai) [inline apps/host/src/components/common/multi-select/multi-select.tsx:66]: overflowY: 'auto'

File: package.json [hunk 2]
```diff
@@ -22,5 +22,8 @@
   "workspaces": [
     "apps/*",
     "shared/*"
-  ]
+  ],
+  "dependencies": {
+    "react-select": "^5.8.0"
```
Comment (sadanandpai) [inline package.json:27]: to be added to Host app

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**